styles.css

```
@@ -0,0 +1,290 @@
+*{
+  font-family: 'GTWalsheim','Didact Gothic' , sans-serif;
+}
+
+body{
+  margin: 0;
+}
+
+/*--------------------------------seccion NAV---------------------------------*/
+
+
+.mainNav{
```

@edwartexe do not write class names in that way (mixing uppercase and lowercase). If you need to separate words, use hyphens. Also, write a blank space between the rule and the bracket. In that way, you can avoid deleting brackets by mistake.
```suggestion
.main-nav {
```